src/_P039_Thermosensors.ino

```diff
@@ -467,23 +485,27 @@ boolean Plugin_039(byte function, struct EventStruct *event, String& string)
             addFormNote(F("Set Resistor Type for MAX31865"));
           }
           {
-            const String Coptions[2] = {F("2-/4-wire"), F("3-wire")};
+            const String Coptions[2] = {F("2-/4"), F("3")};
             const int CoptionValues[2] = {0, 1};
             addFormSelector(F("Connection Type"), F("P039_contype"), 2, Coptions, CoptionValues, P039_CONFIG_4);
+            addUnit(F("wire"));
             addFormNote(F("Set Connection Type for MAX31865"));
           }
           {
-            const String FToptions[2] = {F("60 Hz"), F("50 Hz")};
+            const String FToptions[2] = {F("60"), F("50")};
             const int FToptionValues[2] = {0, 1};
             addFormSelector(F("Supply Frequency Filter"), F("P039_filttype"), 2, FToptions, FToptionValues, P039_RTD_FILT_TYPE);
+            addUnit(F("Hz"));
             addFormNote(F("Set filter frequency for supply voltage. Choose appropriate to your power net frequency (50/60 Hz)"));
           }
           {
-            addFormNumericBox(F("Reference Resistor [OHM]"), F("P039_res"), P039_RTD_RES, 0);
+            addFormNumericBox(F("Reference Resistor"), F("P039_res"), P039_RTD_RES, 0);
+            addUnit(F("Ohm"));
             addFormNote(F("Set reference resistor for MAX31865. PT100: typically 430 [OHM]; PT1000: typically 4300 [OHM]"));
           }
           {
-            addFormFloatNumberBox(F("Offset [K]"), F("P039_offset"), P039_RTD_OFFSET, -50.0f, 50.0f, 2, 0.01f);
+            addFormFloatNumberBox(F("Offset"), F("P039_offset"), P039_RTD_OFFSET, -50.0f, 50.0f, 2, 0.01f);
+            addUnit(F("K"));            
             addFormNote(F("Set Offset [K] for MAX31865. Valid values: [-50.0...50.0 K], min. stepsize: [0.01]"));
```

Comment:
I know it is correct, no need to change it, but my physics senses are tingling, to see 'negative' Kelvin. ;)

Reply:
For some reason ( physics teacher, prof !?)  i kept in mind Kelvin as the SI unit for temperature delta values. Splitting the sign information from the temperature delta value with a seperate checkbox would be less intuitive IMHO.

Reply:
Delta-T is indeed very often (always?) noted in K, so therefore it is correct.
It was just something I had to read twice as my internal parser placed a warning there when reading a negative sign combined with Kelvin :)
That's all.